Create ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md (#2355)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1020-16-1-global-roles-application-and-microservice-view-6855-graft-release-cd-6882.md

---
date: ""

> title: global roles application and microservice view (#6855) [GRAFT][release/cd] (#6882)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Global roles can now view applications and microservices
```

Reviewer: ```suggestion
title: Application access section of each Global role will only show Applications and not Microservices
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57710
version: 1020.16.1
---

> global roles application and microservice view (#6855) [GRAFT][release/cd] (#6882)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, users with global roles were unable to view applications and microservices in the Cumulocity IoT platform. This change now allows users with global roles to view applications and microservices, providing them with a more comprehensive overview of the platform's components. This improvement impacts all users with global roles, enabling them to access and monitor applications and microservices without requiring additional permissions.
```

Reviewer: ```suggestion
Previously under Application access of each global role, Applications and Microservices used to show up, but now the improvement has been made to show only Applications  under "Application access" and not Microservices
```

> global roles application and microservice view (#6855) [GRAFT][release/cd] (#6882)

Reviewer 2: ```suggestion
Previously, the **Application access** list of a global role showed both applications and microservices. This has been changed to only show applications in the *Application access** list.
```
